tests/platform_tests/test_advanced_reboot.py

```diff
@@ -63,14 +68,28 @@ def test_fast_reboot_from_other_vendor(duthosts,  rand_one_dut_hostname, request
     advancedReboot.runRebootTestcase()
 
 @pytest.mark.device_type('vs')
-def test_warm_reboot(request, get_advanced_reboot, verify_dut_health,
+def test_warm_reboot(request, get_advanced_reboot, verify_dut_health, tbinfo,
     advanceboot_loganalyzer, capture_interface_counters):
     '''
     Warm reboot test case is run using advacned reboot test fixture
 
     @param request: Spytest commandline argument
     @param get_advanced_reboot: advanced reboot test fixture
     '''
+    if "dualtor" in tbinfo["topo"]["name"]:
+        duthosts = kwargs['duthosts']
+        tbinfo = kwargs['tbinfo']
+        rand_selected_interface = kwargs['rand_selected_interface']
+        toggle_simulator_port_to_upper_tor = kwargs['toggle_simulator_port_to_upper_tor']
+        toggle_all_simulator_ports = kwargs['toggle_all_simulator_ports']
+
+        toggle_all_simulator_ports(LOWER_TOR)
```

Comment:
Why are we toggling all ports? We need distributed active/standby mux, right?

Reply:
We are toggling all ports to one side and toggling back half of the mux ports to ensure we always keep a mixed mux